docs/website/docs/user-guides/v3-migration-guide.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Migrate from v2 to v3
+sidebar_position: 2
+---
+
+### Migrate existing odo component from v2 to v3
+If you have created an odo component using odo v2, this section will help you move it to use odo v3.
```

Comment:
```suggestion
If you have created an odo component using odo v2, this section will help you move it to use odo v3.

```

Reply:
add newline :)

Reply:
Actually, this may be better as you are just rewording the title?

```suggestion
Before migrating to odo v3 you must delete your application and re-run it:
```

Reply:
I don't understand what exactly you are suggesting here. Replacing the current statement with `Before migrating to odo v3 you must delete your application and re-run it:` is not making sense to me.